Add document for new source cloudPcCrossCloudGovernmentOrganizationMapping and related APIs (#16001)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/cloudpccrosscloudgovernmentorganizationmapping.md

---
title: "cloudPcCrossCloudGovernmentOrganizationMapping resource type"

> description: "Represents Cloud PC organization mapping between public cloud organization and US government cloud organization."

Reviewer: ```suggestion
description: "Represents a Cloud PC organization mapping between a public and US government cloud organizations."
```

author: "SuyliuMS"
ms.localizationpriority: medium
ms.prod: "cloud-pc"
doc_type: resourcePageType
---

# cloudPcCrossCloudGovernmentOrganizationMapping resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents Cloud PC organization mapping between public cloud organization and US government cloud organization.

Reviewer: ```suggestion
Represents a Cloud PC organization mapping between a public and US government cloud organizations.
```

> For GCC (US Government Community Cloud) customers, the Azure Active Directory for tenant is in public cloud, but the Azure resources and Windows 365 Cloud PCs are in US government cloud. This mapping is required for customer administrators to setup & config Windows 365 and for the end users at GCC customers to access their Windows 365 Cloud PCs. The setup and maintenance of the tenant mapping must be done while maintaining the security and compliance requirements for the FedRAMP certification and onboarding to the US Government cloud.

Reviewer: ```suggestion
For GCC (US Government Community Cloud) customers, the Azure Active Directory for the tenant is in a public cloud, but the Azure AD resources and Windows 365 Cloud PCs are in the US government cloud. Therefore, tenant mapping should be set up and maintained while updating the security and compliance requirements for the FedRAMP certification and onboarding to the US Government cloud. Tenant mapping is required for customer administrators when setting up & configuring Windows 365 and for GCC end users accessing their Windows 365 Cloud PCs.
```

## Methods
|Method|Return type|Description|
|:---|:---|:---|
|[Create cloudPcCrossCloudGovernmentOrganizationMapping](../api/cloudpccrosscloudgovernmentorganizationmapping-post.md)|[cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md)|Create a new [cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md) object.|
|[Get cloudPcCrossCloudGovernmentOrganizationMapping](../api/cloudpccrosscloudgovernmentorganizationmapping-get.md)|[cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md)|Read the properties and relationships of a [cloudPcCrossCloudGovernmentOrganizationMapping](../resources/cloudpccrosscloudgovernmentorganizationmapping.md) object.|

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |id|String|The tenant ID of the GCC (US Government Community Cloud) tenant in public cloud. Inherited from [entity](../resources/entity.md).|

Reviewer 2: Could you help remove the `Inherited from [entity](../resources/entity.md).`?

> |organizationIdsInUSGovCloud|String collection|The ID of the tenant in Azure Government corresponding to the GCC (US Government Community Cloud) tenant in public cloud. We only support 1:1 mapping for now, so this collection can only contains 1 tenant ID.|

Reviewer: Could we update the first sentence: Azure Government ____? 
My suggestion for the second sentence: We only support 1:1 mapping for now, so this collection can only contain one tenant ID.

Author: Fixed

> |organizationIdsInUSGovCloud|String collection|The ID of the tenant in Azure Government corresponding to the GCC (US Government Community Cloud) tenant in public cloud. We only support 1:1 mapping for now, so this collection can only contains 1 tenant ID.|

Reviewer: ```suggestion
|organizationIdsInUSGovCloud|String collection|The tenant ID in the Azure Government cloud corresponding to the GCC (US Government Community Cloud) tenant in the public cloud. Currently, we support 1:1 mappings, so this collection can only contain one tenant ID.|
```

Author: Fixed

---
title: "cloudPcCrossCloudGovernmentOrganizationMapping resource type"

> description: "**TODO: Add Description**"

Reviewer 3: Please update the description here.

author: "SuyliuMS"
ms.localizationpriority: medium
ms.prod: "cloud-pc"
doc_type: resourcePageType
---

# cloudPcCrossCloudGovernmentOrganizationMapping resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Represents Cloud PC organization mapping between public cloud organization and US government cloud organization.
For GCC (US Government Community Cloud) customers, the Azure Active Directory for tenant is in public cloud, but the Azure resources and Windows 365 Cloud PCs are in US government cloud. This mapping is required for customer administrators to setup & config Windows 365 and for the end users at GCC customers to access their Windows 365 Cloud PCs. The setup and maintenance of the tenant mapping must be done while maintaining the security and compliance requirements for the FedRAMP certification and onboarding to the US Government cloud.

> Inherits from [entity](../resources/entity.md).

Reviewer 3: I suppose we don't need this line.
